.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,20 @@
+# <a name='#workflow'>Workflow</a>
+
+When working on an issue for the Ignite UI Angular Samples, you need to be aware of and to follow a correct status workflow. We have created a number of status labels in order to communicate well what the current status of a single issue/pull request is. The statuses are as follows:
+
+## Development - applicable to issues and pull requests
+1. `status: in-review` this is the initial status of an issue. If the label is not placed, go ahead and place it.
+2. `status: in-development` this is the status once you start working on an issue. Assign the issue to yourself if it hasn't been assigned already and remove the previous status and assign it an in development status.
```

Comment:
This could be improved a little bit - "assign it an in development status". Maybe "assign `status: in-development`".